sources/packages/backend/libs/services/src/application/application.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from "@nestjs/common";
+import { Repository } from "typeorm";
+import { Application } from "@sims/sims-db";
+import { InjectRepository } from "@nestjs/typeorm";
+
+/**
+ * Service layer for Application.
+ */
+@Injectable()
+export class ApplicationService {
```

Comment:
Are services like these that we should follow the same that we did for the `note.shared.service.ts` adding the "shared" to the file name and class name?